userappwithbroker/build.gradle

```diff
@@ -16,8 +16,8 @@ allprojects {
     }
 }
 
-def adalVersion = "4.8.6"
-def commonVersion = "17.1.0"
+def adalVersion = "4.+"
```

Comment:
specific versions.

Reply:
Talked with Amit offline; keeping these to +.